packages/backend/src/scheduler/SchedulerTask.ts

```diff
@@ -620,6 +620,16 @@ export const uploadGsheetFromQuery = async (
         jobId,
         scheduledTime,
     };
+
+    const analyticsProperties: DownloadCsv['properties'] = {
+        jobId,
+        userId: payload.userUuid,
+        organizationId: payload.organizationUuid,
+        projectId: payload.projectUuid,
+        fileType: SchedulerFormat.GSHEETS,
+        storage: s3Service.isEnabled() ? 's3' : 'local',
```

Comment:
This storage doesn't make sense with gsheet, we should probably make this storage optional (storage?)

Reply:
sounds good! nice 👍